templates/travis_notify.j2

```diff
@@ -0,0 +1,4 @@
+@{{ submitter }} This PR contains references to travis-ci.org. Please rebase your branch and push a new commit. [rebase instructions](http://docs.ansible.com/ansible/dev_guide/developing_rebasing.html)
```

Comment:
Is having a custom message for Travis going to cause a problem when you add one for needs_rebase? Won't you end up with both messages added to the comments?